conventions/documentation/actionmenus.md

```diff
@@ -0,0 +1,41 @@
+---
+parent: Conventions
+---
+
+# Action menus
+
+## Kebab and resource ‘Actions’ menus
+
+- List views of resources have a “kebab” menu that matches the resources’ “Actions” menu in their details view.
+
+![action menus kebab](../images/action-menus-kebab.png)
+
+![action menus actions](../images/action-menus-actions.png)
+
+Ordering of actions in the menu should follow the following pattern:
+- Unique actions to that resource should appear above default resource actions
```

Comment:
Can we provide some general guidance here regarding the ordering of those unique actions?  Maybe primary/most frequently used actions are listed first, and followed by others in alphabetical order? @itsptk

Reply:
I added mention that these unique actions should be sorted by most commonly used first, thanks Serena.